index.js

```diff
@@ -834,7 +842,36 @@
                 this.update();
             }
         },
+        
+        /**
+         * Hides offline messaging for a fullscreen game only experience.
+         */
+        setArcadeMode() {
+            document.body.classList.add(Runner.classes.ARCADE_MODE);
+            this.setArcadeModeContainerScale();
+        },
 
+        /**
+         * Sets the scaling for arcade mode.
+         */
+        setArcadeModeContainerScale() {
+            const windowHeight = window.innerHeight;
+            const scaleHeight = windowHeight / this.dimensions.HEIGHT;
+            const scaleWidth = window.innerWidth / this.dimensions.WIDTH;
+            const scale = Math.max(1, Math.min(scaleHeight, scaleWidth));
+            const scaledCanvasHeight = this.dimensions.HEIGHT * scale;
+            // Positions the game container at 10% of the available vertical window
+            // height minus the game container height.
+            const translateY = Math.ceil(Math.max(0, (windowHeight - scaledCanvasHeight -
+                                                      Runner.config.ARCADE_MODE_INITIAL_TOP_POSITION) *
+                                                  Runner.config.ARCADE_MODE_TOP_POSITION_PERCENT)) *
+                  window.devicePixelRatio;
+
+            const cssScale = scale;
+            this.containerEl.style.transform =
+                'scale(' + cssScale + ') translateY(' + translateY + 'px)';
+        },
+        
```

Comment:
Also, could you describe why you made these changes? It worked fine.

Reply:
I mean the entire index.js file.

Reply:
I added back the arcade mode, and because there are a lot of details I slowed everything down